index.js

```diff
@@ -116,8 +116,15 @@ const RainbowMessage = ( string ) => {
  * @returns {string}             - The HTML to send back to the user
  */
 const GenerateHTML = ( url, query, endpoint, templateDir, { data, variables } = SETTINGS.sass ) => {
+	// Change endpoint to template location, remove any ../
+	const cleanURL = url.replace( /\.\.\//g, '' ).replace( endpoint, templateDir );
+
 	// Location of the index.html file relative to URL.
-	const templateLocation = `${ url.replace( endpoint, templateDir ) }/index.html`;
+	const templateLocation = `${ cleanURL }/index.html`;
```

Comment:
I think we should put this into `SETTINGS.templateLocation`

Reply:
We decided to leave as is.